HabboHotel/Users/Ignores/IgnoresManager.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Dapper;
+using Plus.Database;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Threading.Tasks;
+
+namespace Plus.HabboHotel.Users.Ignores;
+
+public sealed class IgnoresManager : IIgnoresManager
+{
+    private readonly List<int> _ignoredUsers;
```

Comment:
You cannot Dependency inject user specific services right now. All services are global, so you cannot do this. It would just replace the data in ignored users upon login.